cocos/asset/assets/simple-texture.ts

```diff
@@ -141,8 +141,28 @@ export class SimpleTexture extends TextureBase {
      * @param source @en The source image or image data. @zh 源图像或图像数据。
      * @param level @en Mipmap level to upload the image to. @zh 要上传的 mipmap 层级。
      * @param arrayIndex @en The array index. @zh 要上传的数组索引。
+     * @deprecated since v3.9, please use `uploadData()` instead.
```

Comment:
The comment is not clear. Both the function names are `uploadData`.

Reply:
OK